src/runtime/composables/usePortal.ts

```diff
@@ -0,0 +1,24 @@
+import { inject, provide, computed, type Ref, type InjectionKey } from 'vue'
+
+export const portalTargetInjectionKey: InjectionKey<Ref<string | HTMLElement>> = Symbol('nuxt-ui.portal-target')
+
+export function usePortal(portal: Ref<string | HTMLElement | boolean | undefined>) {
+  const portalTarget = inject(portalTargetInjectionKey, undefined)
+
+  const to = computed(() => {
+    if (typeof portal.value === 'string' || portal.value instanceof HTMLElement) {
```

Comment:
@menthol It seems `instanceof HTMLElement` breaks the build, can we check on `typeof portal.value === 'boolean'` instead? 🤔

Reply:
We can remove the HTMLElement check, was only for feature parity with Radix.

string and boolean work perfectly alone

Reply:
If the only problem is this `instanceof`, we can invert the condition